newproject01/newproject01/q1.c

```diff
@@ -0,0 +1,67 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#define N 8
+
+isNumberInOrder(int[], int, int); /*Declaration of a function*/
+
+int main1()
```

Comment:
it is usually better to create two different projects inside Visual Studio, and that way you won't need to do that main1 thing :)

Reply:
I thought about it, but if have 1 project with 3-4 parts, what should I do?